mmcv/runner/epoch_based_runner.py

```diff
@@ -149,14 +149,13 @@ def save_checkpoint(self,
                 Defaults to True.
         """
         if meta is None:
-            meta = dict(epoch=self.epoch + 1, iter=self.iter)
-        elif isinstance(meta, dict):
-            meta.update(epoch=self.epoch + 1, iter=self.iter)
-        else:
+            meta = {}
+        elif not isinstance(meta, dict):
             raise TypeError(
                 f'meta should be a dict or None, but got {type(meta)}')
         if self.meta is not None:
```

Comment:
Suggest to add comments and refer to this PR in case developers miss this important point when modifying the logic.

Reply:
Done. Feel free to change or add other comments.